apps/client/src/components/Questions.js

```diff
@@ -171,7 +171,7 @@ const Questions = ({
     // Track active questions
     if (isQuestionSectionActive) {
       matomoTrackEvent({
-        action: checker.stack[questionIndex].text,
+        action: checker.stack[questionIndex]?.text || "unknown question",
```

Comment:
This is weird IMHO. Can you add this comment above?: `@TODO: make sure the 'questionIndex' is always on the stack`

Reply:
I've added this to the frontend board. We can pick this up.